sdk/storage_blobs/examples/blob_01.rs

```diff
@@ -22,18 +23,11 @@ async fn main() -> azure_core::Result<()> {
     let container_client = storage_client.as_container_client(&container_name);
     let blob_client = container_client.as_blob_client("SorgeniaReorganizeRebuildIndexes.zip");
 
-    let _res = container_client
-        .list_blobs()
-        .include_copy(true)
-        .include_deleted(true)
-        .include_metadata(true)
-        .include_snapshots(true)
-        .include_uncommitted_blobs(true)
-        .execute()
-        .await?;
-
-    let result = blob_client.get().execute().await?;
-
+    // only get the first 8k chunk
+    let result = Box::pin(blob_client.get().stream(1024 * 8))
```

Comment:
Is the `Box::pin` necessary here? I believe that `Pageable` streams are already `Unpin`.

Reply:
This will get addressed in a follow-on PR moving to Pipelines (and therefore Pageable)